ccpp/data/GFS_typedefs.meta

```diff
@@ -4610,6 +4650,12 @@
   units = flag
   dimensions = ()
   type = logical
+[do_spp]
+  standard_name = flag_for_stochastic_spp_option
```

Comment:
standard_name suggestion: do_stochastically_perturbed_parameterization

The CCPP community has settled on "do_..." rather than flag. This hasn't propagated to the UFS yet, but we can start using it. See, e.g. https://github.com/ESCOMP/CCPPStandardNames/blob/main/standard_names.xml

Reply:
Updated.  Thanks!

Reply:
@JeffBeck-NOAA it seems to me the name is not updated to "do_stochastically_perturbed_parameterizations"?

Reply:
Updated.  Thanks!